packages/components/src/__future__/Tag/Tag.module.scss

```
@@ -0,0 +1,79 @@
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/typography";
+
+.container {
+  display: inline-flex;
+  align-items: center;
+  padding: 0.3rem $spacing-8 0.3rem $spacing-8;
```

Just checked the designs, and it should actually be 4px on the y-axis

```suggestion
  padding: $spacing-4 $spacing-8;
```

https://www.figma.com/file/hANEprZKom5Br1IBXhE8mr/%F0%9F%A7%AA-Kaizen-Labs?node-id=28615%3A183611&mode=dev